src/vue/pages/VerificationCorporate.vue

```diff
@@ -261,13 +257,22 @@ export default {
           this.kvEntryCorporateRoleId
         )
         await Api.api.postOperations(operation)
+        if (
+          this.kycState === REQUEST_STATES_STR.pending ||
+          this.kycState === REQUEST_STATES_STR.approved ||
+          this.kycState === REQUEST_STATES_STR.rejected
+        ) {
+          Bus.success('verification-form.request-updated-msg')
+        }
```

Comment:
what the message shown to the user if the request has been sent for the first time or after being rejected permanently?

Reply:
`Request was successfully updated` for all cases when the user updates the previous KYC. For new KYC requests we saying nothing as in the past.

Reply:
Let’s give a feedback for the new request also